services/web/server/tests/unit/with_dbs/docker-compose-devel.yml

```diff
@@ -62,7 +62,16 @@ services:
     image: rediscommander/redis-commander:latest
     restart: always
     environment:
-      - REDIS_HOSTS=resources:redis:6379:0,locks:redis:6379:1,validation_codes:redis:6379:2,scheduled_maintenance:redis:6379:3,user_notifications:redis:6379:4,announcements:redis:6379:5
+      - >-
+        REDIS_HOSTS=
+        resources:redis:6379:0:${TEST_REDIS_PASSWORD},
```

Comment:
We tend to have the same env var names for both tests and production. This makes it harder to maintain

Reply:
Not sure how to read this comment of yours, is this a change request? can you comment on my response to sylvain please? thx (see screenshot) 
![image](https://github.com/user-attachments/assets/3fa4b244-9dc4-4478-a0be-cdabdd0d36d6)

Reply:
Sorry, I missed that answer :-) Aha, your PR makes sense now. 

This is pretty old and today I would do it differently. I will create a maintenance PR to follow up on yours and remove all the TEST_ variables. In principle there is no reason to have different environs since we can always patch them

thx